assets/css/editor.scss

```diff
@@ -60,3 +60,26 @@
 	margin: 0;
 	padding: 0;
 }
+
+.theme-twentytwenty {
+	.wp-block {
+		h2.wc-block-grid__product-title {
+			@include font-size(large);
+		}
+
+		.wc-block-grid__product-price,
+		.wc-block-active-filters__title,
+		.wc-block-attribute-filter__title,
+		.wc-block-price-filter__title {
+			@include font-size(regular);
+		}
+
+		.wc-block-grid__product-add-to-cart.wp-block-button {
+			.wp-block-button__link {
+				@include font-size(small);
+				padding: em($gap-small);
+				text-decoration: none;
+			}
+		}
```

Comment:
Do you think we can remove this ruleset? The one in `style.scss` will already have effect in the editor, so it should be fine defining all styles there.

Reply:
> Do you think we can remove this ruleset? The one in `style.scss` will already have effect in the editor, so it should be fine defining all styles there.

Thanks for pointing out that `style.scss` will also affect the editor styles. I removed the section that you mentioned, and the styles in the editor remained the same.